build.go

```diff
@@ -92,6 +91,18 @@ func Build(
 			}
 		}
 
+		nugetCache, err := context.Layers.Get("nuget-cache")
+		if err != nil {
+			return packit.BuildResult{}, err
+		}
+
+		nugetCache.Cache = true
```

Comment:
For my own understanding why is the NuGet cache not needed at launch? Would the app not need access to it?

Reply:
To the best of my understanding is that these are cached and then used during compilation in the same way that go modules are used. So once the application has been compiled then they are no longer needed.